deps/patches/mbedtls.patch

```diff
@@ -0,0 +1,27 @@
+diff --git include/mbedtls/config.h include/mbedtls/config.h
```

Comment:
can we call this file something a little more specific? `mbedtls-config.patch` or similar?